modules/ROOT/pages/fix-instance-conformance-issues.adoc

```diff
@@ -0,0 +1,63 @@
+= Finding and Fixing Conformance Issues in API Instances
+
+After API Governance identifies conformance issues in your API instances, you can view the governance validation report details to get the information you need to fix the issues. After you identify what needs to be fixed, go to the instance in API Manager and fix the issue. If the issue is an instance-related issue in the specification, use your API editor, such as API Designer, to fix the specification.
+
+// Screenshot of a Conformance Report page with callouts 
+
+include::_partials/api-gov-apim-governance-report.adoc[]
+
+== View Conformance Issue Details in the Governance Report
+
+To view instance conformance issues in the governance validation report:
+
+. Navigate to the report.
+* In API Governance:
++
+.. In the API Governance console, select the *Governed APIs* tab.
+.. If the *Instances* column has a *Nonconformant* badge, click *View Report* for information about the conformance issue.
+* In API Manager:
+.. Select the *Conformance Report* page for the API that is marked as nonconformant.
+. View nonconformance details for each ruleset:
++
+* Click the icon for the nonconformance severity to see details for issues for that severity level. 
+* Click *View Details* to see the full breakdown of conformance issues.
+
+== View Rule Details in the Rulesets
+
+If the nonconformance messages in the governance validation report don't have enough detail to help you fix the issues, you can view more detailed information in the rulesets. 
+
+To view rules details in rulesets:
+
+. In Exchange, open the ruleset. 
+. View the documentation for the ruleset.
+. To identify the conformance issue, view descriptions and messages for the rules to which your API doesn't conform.
+
+// Screenshot of a section of the ruleset with a description of the properties 
+
+include::_partials/api-gov-apim-ruleset-fix-tips.adoc[]
+
+== Example Rules and Recommended Fixes
+
+The following table shows some example rules and the actions that you can take to fix conformance to those rules.
+
+[cols="3a,3a,3a",frame=none]
+|===
+
+|Rule
+|Scope
+|Recommended Fix
+|$BEFORE policies should be configured before $AFTER policies
```

Comment:
```suggestion
|Configure $BEFORE policies before $AFTER policies
```

Reply:
(That's probably wrong - I was reading it as a 2-column table.)

Reply:
This text is in the ruleset itself, so the text needs to match.